src/components/Layouts.jsx/MainLayout.jsx

```
@@ -324,40 +327,42 @@ const MainLayout = (props) => {
+                contentType.toLowerCase() !== "word" &&
+                startShowCase && (
+                  <Box
+                    position={"absolute"}
+                    top={20}
+                    left={20}
+                    justifyContent={"center"}
+                  >
+                    <Box display={"flex"}>
+                      {[...Array(Math.max(0, redLivesToShow) || 0).keys()]?.map(
+                        (elem) => (
+                          <HeartRed />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)